app/src/main/java/com/fs_sournary/weather/utils/Constant.java

```diff
@@ -0,0 +1,13 @@
+package com.fs_sournary.weather.utils;
+
+/**
+ * Created by fs-sournary.
+ * Date on 11/17/17.
+ * Description:
+ */
+
+public class Constant {
+
+    public static String CURRENT_FRAGMENT_TAG = "current_fragment";
```

Comment:
ERROR: [MALICIOUS_CODE][MS_SHOULD_BE_FINAL] com.fs_sournary.weather.utils.Constant.CURRENT_FRAGMENT_TAG isn't final but should be
http://findbugs.sourceforge.net/bugDescriptions.html#MS_SHOULD_BE_FINAL